tests/mainnet/history/headers_with_proof/22162263.yaml

```diff
@@ -0,0 +1,5 @@
+content_value: "0x080000006f020000f90264a01b06c8842aae3f67bbc5b70ba4b6a1d31b6bceb703124439941e2fea04a5ffeba01dcc4de8dec75d7aab85b567b6ccd41ad312451b948a7413f0a142fd40d49347944838b106fce9647bdf1e7877bf73ce8b0bad5f97a087c12abfde54386f5e268789195691c7be3e119608df6567030f7329d87c3240a0157b7c9161c55469dc7b623d57ae5ffc7f201754fd3dbaf30763bf347b987183a01264e71f9a4239f883de8571f6f36850b49c57043c8d166d0b61a3d745eb8ca5b90100dfef73775ff6dfbb57bfbbbfeffff6afbfff77f96ef3bffcef3fe7df6efbed56dec7dff7ffff7ffffffeffbbbfff47feffb96f3fdebbefddeff7fef5e97fbcf7ef7fb7bff77fef9a6eee71bffff6f9fb71fdef3d4eddafef7cff3db3bffd16edfef8bdfffafff7fbe3edfff7fffbfffff67ce7eedfbecce7ffafffb73dffdfff7ecb76ffffb977feebfdfffaaffffed23f7fef3ffdfd6bfffffefedb757bdbf9ef3cbffbdff6ff9fffdbfee7ffffe6df9fffd1ee77ff5ff9dffe5df41fbfdfcf9e355e7f7dbd2affd7eadfdfb29eafeffffebbc7fffdf23ff7cdd7ebeef5fffffb7bbfff6ffdf1ff8fd7f7b5bf77feffa7fdfd79fffde9f8dfddf6befeebffb3808401522b578402243dd383f448488467e9b04b98546974616e2028746974616e6275696c6465722e78797a29a0645c10a124c50017fc7da9c36fe40d4c736aa1644bd45807aa86b27775fbebce8800000000000000008416868c9da04055697b676be136095f90bd5d08290ded049bcd81b0e4e1991783ab57a8a242830c00008401aa0000a0663c1c7324b757120ded349256377339721f588378e544a35acc524df1edcf7f663c1c7324b757120ded349256377339721f588378e544a35acc524df1edcf7f8b7a8a7628be95fcddb60ee03e9cae87ec3d4f0879b0b1572739ea8ab7d8b53a60cf79e426d83e9e9b3a69fa37013061d436769f633b0b13f3e7ebe3a659abfdc814f1c66c38f5f71377d617ebcab46ac2072e25b7a6df008fc9fcd2f2397b739cd4ed4355d25562c47b862d56c4a72a5a1556bb5b1e5f264b5b8146cf6d2fe4606a5a8c25047dbcbc49046a942d06bd26410acfb9003aa61bc3a3b397ef640dc6acd6acb6dd81c816fd29c0f0ec699b8821d51b93beb9bfabc8b01329472ec68daf2e3a815f8857ec56773400f06155db31de3eb23a9aebebe6a98a08346fdb0c44b033f3788a949910463d4418e4fbe805bcaf2d38bf717e9d36aa12b6f424c0ec6d13181f60d5d9aba44056b484a142a41cd327f30199e5d579dd63c371db9b0c9d881805365cbb27ceab374e9bae594290743a9aba318a67f0f82abcda27dd7a8e2fbbe4df2730c37e3026b4161b28cbb4a4586cec606fcaa61389969d639f9ff19845ba987d672380b1a010edae98df5b3416f9b84e2d7aa3827a41f1c3452eb0df043bab8ec645217e50a04ff6bc18ba9898468643e44a186b97bb94f1cc010000ff9fad00000000000782512b0075cdc017383c74639c9dfe6b14a5a4244c2683fef4a491ecc0a730a91e24d4b51f23b5dc6ae1e27342533fa7dbf9394ce4808db64ad37a063c62678553b496b975dcb747c0e6e496607c41e658f309ed8307bdd145a494c946f39819f4868e7b08709b8165dab5fa3fdd28c955217619aa081829eec5395a235825e830a8772ce297251a628f267934b21a7bd94b6e6415136b6518b5536ae877dcc2399738abb3bed5d2d2b39f9c6d732f6492d62c4d4669ea3d76a7e46410e9481dd63e221528163153337217e41d81e8afe876fecf9e20e40e5e743f48197981db56114e00fdd4c1f85c892bf35ac9a89289aaecb1ebd0a96cde606a748b5d71c3a54c84046c5b2644325e0171426559df7fbc0624efa9820e7866a04ba1ad850000000000000000000000000000000000000000000000000000000000000000f5a5fd42d16a20302798ef6ed309979b43003d2320d9f0e8ea9831a92759fb4be55723512d586a3fc7c83dfd06cafe74472a91428fa1fa59219e487bec6447ae"
```

Comment:
Probably correct, but I cannot verify as in fluffy we have already moved to using `Vector`  instead of `List`.